packages/taquito/src/parser/michel-codec-parser.ts

```diff
@@ -32,6 +34,25 @@ export class MichelCodecParser implements ParserProvider {
         parsedParams.code = await this.formatCodeParam(params.code);
         if (params.init) {
             parsedParams.init = await this.formatInitParam(params.init);
+        } else if (params.storage) {
+            const storageType = (parsedParams.code as Expr[]).find((p): p is Prim => ('prim' in p) && p.prim === 'storage');
+            if (storageType?.args === undefined) {
```

Comment:
we might be able to shorten this to just 
`if (!storageType?.args)`

Reply:
Thanks, I made the change, it is cleaner like that.